src/commands/frameworks-stacks-delete.ts

```diff
@@ -0,0 +1,24 @@
+import { Command } from "../command";
+import { Options } from "../options";
+import { needProjectId } from "../projectUtils";
+import { FirebaseError } from "../error";
+import * as gcp from "../gcp/frameworks";
+import isEmpty from "lodash/isEmpty";
```

Comment:
nit* There is general tendency in this repo to avoid using lodash when it's not absolutely necessary.

```
if (!stackId) {
}
```

should be equivalent